apps/mobile/app.config.ts

```diff
@@ -1,5 +1,5 @@
-const VERSION_CODE = 41
```

Comment:
updating the version in this PR as previous release/42 branch did not want to rebase for some reason